docs/quickstart.md

```diff
@@ -40,72 +42,307 @@
+            "createdOn": 1575485630,
+            "id": "775788068104",
+            "lastModifiedOn": 1575485630,
+            "principalPolicyHash": "\"bc5872b50475b186afea67ff47516a8f\"",
+            "principalRoleArn": "arn:aws:iam::775788768154:role/DCEPrincipal-quickstart"
+        }
+    ]
+    ```
     The account status will initially say `NotReady`. It may take up to 5 minutes for the new account to be processed. Once the account status is `Ready`, you may proceed with creating a lease.
 
-10. Now that your accounts pool isn't emtpy, you can create your first lease using the `dce leases create` command
+## Leasing a DCE Account
 
-```
-➜  ~ dce leases create --budget-amount 100.0 --budget-currency USD --email jane.doe@email.com --principal-id jdoe99
-```
+1. Use the `dce auth` command to authenticate as a non-admin user.
 
-11. Type `dce leases list` to verify that a lease has been created
+1. Now that your accounts pool isn't empty, you can create your first lease using the `dce leases create` command.
 
-```
-➜  ~ dce leases list
-[{"accountId":"555555555555","budgetAmount":100,"budgetCurrency":"USD","budgetNotificationEmails":["jane.doe@email.com "],"createdOn":1572562298,"id":"d326bddf-af36-44d9-bbd0-70b9f7e55356","lastModifiedOn":1572637591,"leaseStatus":"Active","leaseStatusModifiedOn":1572637591,"principalId":"jdoe99"}]
-```
+    ```
+    dce leases create --budget-amount 100.0 --budget-currency USD --email jane.doe@email.com --principal-id jdoe99
+    Lease created: {
+        "accountId": "555555555555",
+        "budgetAmount": 100,
+        "budgetCurrency": "USD",
+        "budgetNotificationEmails": [
+            "jane.doe@email.com"
+        ],
+        "createdOn": 1575490207,
+        "expiresOn": 1576095007,
+        "id": "e501cb86-8317-458b-bdce-d47ab92f86a8",
+        "lastModifiedOn": 1575490207,
+        "leaseStatus": "Active",
+        "leaseStatusModifiedOn": 1575490207,
+        "leaseStatusReason": "Active",
+        "principalId": "jdoe99"
+    }
+    ```
+   
+1. Type `dce leases list` to verify that a lease has been created
 
-12. Access your leased account programmatically via the `dce leases login` command. E.g.
+    ```
+    dce leases list
+   [
+   	{
+   		"accountId": "555555555555",
+   		"budgetAmount": 100,
+   		"budgetCurrency": "USD",
+   		"budgetNotificationEmails": [
+   			"jane.doe@email.com"
+   		],
+   		"createdOn": 1575490207,
+   		"expiresOn": 1576095007,
+   		"id": "e501cb86-8317-458b-bdce-d47ab92f86a8",
+   		"lastModifiedOn": 1575490207,
+   		"leaseStatus": "Active",
+   		"leaseStatusModifiedOn": 1575490207,
+   		"leaseStatusReason": "Active",
+   		"principalId": "jdoe99"
+   	}
+   ]
+    ```
 
-```
-# View your leased account credentials
+1. If we try to login to this leased account, we will receive a permissions error since it is registered under a different user (i.e. `jdoe99` != `quickstartuser`).
```

Comment:
@joshmarsh I don't think we should be creating a lease for a different user at all. That's a bug in our system that you can do that, right?